README.md

```diff
@@ -12,16 +12,24 @@ gem 'puppet_fixtures'
 ```
 
 Now you can use `bundle exec puppet-fixtures`, for example:
-```console
+
+```shell
```

Comment:
This is still console output (note the `$` prefix` and then output)

Reply:
It follows the convention that `$` is a command that the user enters in a non-root shell then the non-prefixed lines are the output. `#` are the commands that are expected to run in a root shell.